src/Files.App/ViewModels/UserControls/Previews/CodePreviewViewModel.cs

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) 2024 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
+using System.Collections.Frozen;
 using ColorCode;
 using Files.App.ViewModels.Properties;
 
 namespace Files.App.ViewModels.Previews
 {
 	public sealed class CodePreviewViewModel : BasePreviewModel
 	{
-		private static readonly Lazy<IReadOnlyDictionary<string, ILanguage>> extensions = new(GetDictionary);
+		private static readonly Lazy<FrozenDictionary<string, ILanguage>> extensions = new(GetDictionary);
```

Comment:
I don't think `Lazy` is necessary, because even without it, the dictionary should be initialized when `ContainsExtension` is called for the first time.

Reply:
Done

Reply:
I think `hotKeys` should remain `ImmutableDictionary` because it could be reinitialized.